docs/tutorials/05-model-monitoring.ipynb

```diff
@@ -64,43 +61,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "tsdb_profile = DatastoreProfileV3io(name=\"v3io-tsdb-profile\")\n",
-    "project.register_datastore_profile(tsdb_profile)\n",
+    "from src.model_monitoring_utils import enable_model_monitoring\n",
     "\n",
-    "stream_profile = DatastoreProfileV3io(\n",
-    "    name=\"v3io-stream-profile\",\n",
-    "    v3io_access_key=mlrun.mlconf.get_v3io_access_key(),\n",
-    ")\n",
-    "project.register_datastore_profile(stream_profile)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "project.set_model_monitoring_credentials(\n",
-    "    tsdb_profile_name=tsdb_profile.name,\n",
-    "    stream_profile_name=stream_profile.name,\n",
-    ")"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "> 2024-10-20 08:49:27,599 [warning] enable_model_monitoring: 'base_period' < 10 minutes is not supported in production environments: {\"project\":\"tutorial\"}\n"
-     ]
-    }
-   ],
-   "source": [
-    "project.enable_model_monitoring(base_period=2)"
+    "# Disabling old model monitoring to update configurations\n",
```

Comment:
If model monitoring was enabled in v1.7.x: disable model monitoring to update the configurations